include/libtorrent/torrent_handle.hpp

```diff
@@ -1265,6 +1281,24 @@ namespace aux {
 		// client builds.
 		std::shared_ptr<torrent> native_handle() const;
 
+		// Returns true if the torrent is active in its session; returns false if
+		// it has been removed.
```

Comment:
it's probably a good idea to specify that "removed" means `session::remove_torrent()` has been called